ui/component/textareaWithSuggestions/view.jsx

```diff
@@ -102,7 +103,15 @@ export default function TextareaWithSuggestions(props: Props) {
   const suggestionTerm = suggestionValue && suggestionValue.term;
   const isEmote = suggestionValue && suggestionValue.isEmote;
   const isMention = suggestionValue && !suggestionValue.isEmote;
-  const invalidTerm = suggestionTerm && isMention && suggestionTerm.charAt(1) === ':';
+
+  let invalidTerm = suggestionTerm && isMention && suggestionTerm.charAt(1) === ':';
+  if (isMention && suggestionTerm) {
+    try {
+      parseURI(suggestionTerm);
```

Comment:
--- just a note ---
I was curious how `parseURI` would perform, since it has been the source of many performance issues and since this will be continuously rendered as the user types.  Seem ok even on debug.  Maybe it was full lbry urls that's slow.

<img width="256" alt="image" src="https://user-images.githubusercontent.com/64950861/151308072-19e5a66c-63b2-4692-a6cb-e038e108b573.png">